### ISSUE TITLE
Display more information about query stages on query details page

### ISSUE THREAD

**Issue by wankunde** [Intel-bigdata/spark-adaptive#85]
SQL in product environment can be very complex and can be divided into multiple query stages.
If a spark job is running slowly, the job details page will only the the DAG of the job,but we need more information for troubleshooting,such as the tables that the job is scanning, the type of the job, and so on. But those message can be found on query details page.

Can we show the query stage that the slow-running job corresponds to ?

And I thind it's better to show the physical tasks which query stage corresponds to on the query details page.

**Comment by wankunde** [comment]
![image](https://user-images.githubusercontent.com/3626747/53626398-92cd1c80-3c40-11e9-93cf-6eb63ab5777f.png)


As shown in the screenshot, physical tasks are divided into multiple query stages. But it still fails to show the relationship between the query stages and the jobs...

**Comment by wankunde** [comment]
![image](https://user-images.githubusercontent.com/3626747/53729052-2015a880-3eaf-11e9-8dbb-204d58e5bbb6.png)

The SparkPlanGraph may be should like this, but the web ui can not display this. 
Strange～～

![image](https://user-images.githubusercontent.com/3626747/53729220-9a462d00-3eaf-11e9-9a7f-5f416d39c57f.png)

**Comment by yucai** [comment]
I think the idea is good, but it is not just an AE issue, the original Spark needs this kind of improvement also.

**Comment by carsonwang** [comment]
yes, it will be good to have this kind of mapping information to simplify the debugging. Does your change work for simple queries?

**Comment by wankunde** [comment]
![image](https://user-images.githubusercontent.com/3626747/53797026-4ea78800-3f70-11e9-885d-ebcbe8742f6c.png)

Now the query stages information can be displayed.

**Comment by wankunde** [comment]
@carsonwang @yucai  I create #89 to display every query stages in the query. Minor UI changes .
